muskathlon/controllers/main.py

```diff
@@ -47,7 +47,7 @@ def my_muskathlon_details(self, registration, **kwargs):
         )
         return request.render("muskathlon.my_details", {"reports": reports})
 
-    @route(["/my", "/my/home"], type="http", auth="user", website=True)
+    @route(["/my", "/my/home"], type="http", auth="user", website=True, cors="*")
```

Comment:
Just out of curiosity, what is `cors` used for?

Reply:
I added it because there are CORS error when loading the page and I found other places where this is done. But it doesn't do anything, so I will remove it